README.md

```diff
@@ -57,17 +50,26 @@ To start Express, run the following from the root directory:
 
 This will run the express server on `http://localhost:3000`
 
-## Run Client-Side Tests
+### 6. Local HTTP-Proxy to Facet Service
+
 
-Run the following from the `spa` directory:
+The URL for the back end Azul Facet Service is configurable from the environment property ```BW_DATA_URL```. This can be set to the url of the server running the back end in the Gruntfile to run locally or on the boardwalk node.js server.
```

Comment:
I would prefer `Azul Webservice` over `Azul Facet Service`, but I don't feel too strongly about it.